janusgraph-hadoop-parent/pom.xml

```diff
@@ -205,7 +205,13 @@
         <dependency>
             <groupId>com.datastax.cassandra</groupId>
             <artifactId>cassandra-driver-core</artifactId>
-            <version>${cassandra-driver.version}</version>
+            <version>3.7.2</version>
```

Comment:
This is because this class still depends on the old driver: https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-hadoop-parent/janusgraph-hadoop-core/src/main/java/org/janusgraph/hadoop/formats/cassandra/CqlBridgeRecordReader.java#L26

Reply:
not sure if we still need that class, if we dont I will just remove it

Reply:
You can delete CqlBridgeRecordReader and remove all explicit tests for it. Basically. call cassandra3